lib/parse.js

```diff
@@ -44,6 +44,114 @@
 
 "use strict";
 
+import {
+    all,
+    characters,
+    configure_error_stack,
+    defaults,
+    find_if,
+    HOP,
+    makePredicate,
+} from "./utils.js";
+import {
+    AST_Accessor,
+    AST_Array,
+    AST_Arrow,
+    AST_Assign,
+    AST_Await,
+    AST_Binary,
+    AST_BlockStatement,
+    AST_Break,
+    AST_Call,
+    AST_Case,
+    AST_Catch,
+    AST_ClassExpression,
+    AST_ConciseMethod,
+    AST_Conditional,
+    AST_Const,
+    AST_Continue,
+    AST_Debugger,
+    AST_Default,
+    AST_DefaultAssign,
+    AST_DefClass,
+    AST_Definitions,
+    AST_Defun,
+    AST_Destructuring,
+    AST_Directive,
+    AST_Do,
+    AST_Dot,
+    AST_EmptyStatement,
+    AST_Expansion,
+    AST_Export,
+    AST_False,
+    AST_Finally,
+    AST_For,
+    AST_ForIn,
+    AST_ForOf,
+    AST_Function,
+    AST_Hole,
+    AST_If,
+    AST_Import,
+    AST_IterationStatement,
+    AST_Label,
+    AST_LabeledStatement,
+    AST_LabelRef,
+    AST_Lambda,
+    AST_Let,
+    AST_NameMapping,
+    AST_New,
+    AST_NewTarget,
+    AST_Node,
+    AST_Null,
+    AST_Number,
+    AST_Object,
+    AST_ObjectGetter,
+    AST_ObjectKeyVal,
+    AST_ObjectProperty,
+    AST_ObjectSetter,
+    AST_PrefixedTemplateString,
+    AST_PropAccess,
+    AST_RegExp,
+    AST_Return,
+    AST_Sequence,
+    AST_SimpleStatement,
+    AST_String,
+    AST_Sub,
+    AST_Super,
+    AST_Switch,
+    AST_SymbolCatch,
+    AST_SymbolClass,
+    AST_SymbolConst,
+    AST_SymbolDeclaration,
+    AST_SymbolDefClass,
+    AST_SymbolDefun,
+    AST_SymbolExport,
+    AST_SymbolExportForeign,
+    AST_SymbolFunarg,
+    AST_SymbolImport,
+    AST_SymbolImportForeign,
+    AST_SymbolLambda,
+    AST_SymbolLet,
+    AST_SymbolMethod,
+    AST_SymbolRef,
+    AST_SymbolVar,
+    AST_TemplateSegment,
+    AST_TemplateString,
+    AST_This,
+    AST_Throw,
+    AST_Token,
+    AST_Toplevel,
+    AST_True,
+    AST_Try,
+    AST_UnaryPostfix,
+    AST_UnaryPrefix,
+    AST_Var,
+    AST_VarDef,
+    AST_While,
+    AST_With,
+    AST_Yield,
+} from "./ast.js";
```

Comment:
Perhaps an `import * as ast from "./ast.js"` would suffice? As far as I know rollup is able to tree-shake wildcard imports.

Reply:
I tried to minimize change of existing lines of code for easier cherry-picking from uglify-js, `import * as ast` will add a lot of edits. Is that phase of the project over? If you don't mind I hope this can be a followup, that will help with reducing conflicts while this is WIP.

Reply:
Awesome, that makes total sense. Was just curious while browsing the repo (I'm not a contributor here :) )

Reply:
I see now I didn't read @fabiosantoscode 's comment, thought he meant the same as @marvinhagemeister 's. Yes it is possible, but eslint rule `no-undef` will let you know as soon as an import is missing. I was thinking that adding `no-unused-vars` would be a good addition but that goes against @fabiosantoscode 's suggestion.